recipe/meta.yaml

```
@@ -1,88 +1,87 @@
 {% set name = "pyg4ometry" %}
-{% set version = "1.0.2" %}
-{% set antlr_version = "4.7.2" %}
+{% set version = "1.3.5" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  - url: https://bitbucket.org/jairhul/pyg4ometry/get/v{{ version }}.tar.gz
-    sha256: 6e2c1224baedbdc681c053b8d281616035ea2118a141018b63a680f924a443d5
-    folder: pyg4ometry
-    patches:
-      - 0001-Fix-setting-includeSearchDirs-and-librarySearchDirs.patch
-      - 0002-Remove-test-only-dependencies.patch
-  - url: https://www.antlr.org/download/antlr-{{ antlr_version }}-complete.jar
-    sha256: 6852386d7975eff29171dae002cc223251510d35f291ae277948f381a7b380b4
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pyg4ometry-{{ version }}.tar.gz
+  sha256: dde0877bc1c214cedaec7b8015d201d215977f2c5c86e307de6f5451219034b0
 
 build:
-  number: 0
   skip: true  # [win or py<38]
-  script:
-    - cd pyg4ometry
-    # Regenerate the antlr parsers so they can be used with a newer version of the runtime
-    - find pyg4ometry/ -name '*.g4' -print -exec java -jar "$SRC_DIR/antlr-{{ antlr_version }}-complete.jar" {} -Dlanguage=Python3 -visitor \;
-    # Update the antlr-python-runtime dependency pin for setuptools
-    - sed -E -i.bak 's@"antlr4-python3-runtime==.+"@"antlr4-python3-runtime=={{ antlr_version }}"@g' setup.py
-    - grep antlr4-python3-runtime=={{ antlr_version }} setup.py
-    # Build and install pyg4ometry itself
-    - {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - pyg4ometry = pyg4ometry.cli:main
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython                                 # [build_platform != target_platform]
-    - pybind11                               # [build_platform != target_platform]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cmake >=3.20
     - make
-    - sed
-    - openjdk
+    - python                              # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - {{ pin_compatible('numpy') }}       # [build_platform != target_platform]
```

if you need numpy to cross-compile, just do
```suggestion
    - numpy                               # [build_platform != target_platform]
```